### ISSUE TITLE
anamon needs to remain python2 compatible

### ISSUE THREAD

**Issue by opoplawski** [cobbler/cobbler#2168]
Because the anamon script runs on the installed system it needs to remain python agnostic.  Commit 946c05df3a15cca70bc73cb4f495e6cd04cc6eb8 changed it to require xmlrpc from python 3.

Also, pre_anamon executes it with:
```
python /tmp/anamon --name "$name" --server "$server" --port "$http_port"
```
but python does not exist on EL8.  I would suggest something like:
```
python=python
[ -x /usr/libexec/platform-python ] && python=/usr/libexec/platform-python
$python /tmp/anamon --name "$name" --server "$server" --port "$http_port"
```

**Comment by Conan-Kudo** [comment]
`python3` exists on both, though, so couldn't we do that?

**Comment by opoplawski** [comment]
Not in the installer environment which in where anamon runs (in %pre).

**Comment by Conan-Kudo** [comment]
Ah, fair point.

**Comment by SchoolGuy** [comment]
@opoplawski Is this the only thing that we need to think of when fixing this or is there more required when fixing the Py2/3 compability with this script?

**Comment by opoplawski** [comment]
I think this is fixed now with 0702a7ba3797e18c250faa61d6f0099ed9450951
